.github/workflows/osx.yml

```diff
@@ -33,6 +30,20 @@ jobs:
       run: |
         make -j 2
         make -j 2 test # quicktests
+    - name: archive error 1
+      uses: actions/upload-artifact@v1
+      if: always()
+      continue-on-error: true
+      with:
+        name: osx-serial-CMakeOutput.log
+        path: CMakeFiles/CMakeOutput.log
+    - name: archive error 2
+      uses: actions/upload-artifact@v1
+      if: always()
+      continue-on-error: true
+      with:
+        name: osx-serial-CMakeError.log
+        path: CMakeFiles/CMakeError.log
```

Comment:
@masterleinad I suggest we just keep this in?

Reply:
Agreed.